lib/real_world/http/mappers/retriever/json.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'json'
+
+module RealWorld
+  module Http
+    module Mappers
+      class Retriever
+        class JSON
```

Comment:
I think it would be a good idea to explain the concept of mappers, retrievers, and so on.

We could write comments above the modules, or maybe explain the whole architecture in in a local `README.md` at `lib/real_world/http`. We could link to the article there, too.

Reply:
I'll open an issue for this one